hack/vendor.sh

```diff
@@ -52,6 +51,10 @@ clone git github.com/miekg/dns 75e6e86cc601825c5dbcd4e0c209eab180997cd7
 clone git github.com/docker/distribution 9ec0d742d69f77caa4dd5f49ceb70c3067d39f30
 clone git github.com/vbatts/tar-split v0.9.11
 
+# get go-zfs packages
+clone git github.com/mistifyio/go-zfs 22c9b32c84eb0d0c6f4043b6e90fc94073de92fa
```

Comment:
It would be nice if they had a release before docker 1.12, but I think we could merge this before that.